content/gocourse-2019-05-melbourne.slide

```diff
@@ -530,17 +530,7 @@ Document them
 - Install [[https://golang.org/doc/install][go 1.12]]
 - Install [[https://github.com/golangci/golangci-lint#install][golangci-lint]]
 
-
-* Repository setup overview
-
-Clone your fork of [[https://github.com/anz-bank/go-course][anz-bank/go-course]]
-
-Create PRs
-
--  from `feature` branch of your own fork (remote `origin`)
--  to `master` on [[https://github.com/anz-bank/go-course][anz-bank/go-course]] (remote `upstream`)
-
-* Repository setup details
+* Repository setup
```

Comment:
I'm confused, why does this show up?
It's already been added (see deployed master at http://go-course.org/gocourse-2019-05-melbourne.slide#45).
Have you done:
```bash
git checkout master
git pull upstream master
git checkout content/update-slide-errors
git rebase -i master
git push -f
```
?

Also true for a few edits below....

Reply:
Good catch. This was added by @anzboi in commit 85268b54. Not sure how these changes made it into this PR.

Reply:
Arrgghh.. Too many updates between my commits. I'll do that and re-push. Thanks for that